config/initializers/health_monitor.rb

```diff
@@ -1,20 +1,26 @@
 # frozen_string_literal: true
-HealthMonitor.configure do |config|
-  config.cache
-  config.redis
 
-  # Make this health check available at /health
-  config.path = :health
+Rails.application.config.to_prepare do
```

Comment:
This is explained: https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#use-case-1-during-boot-load-reloadable-code